spaceship/lib/spaceship/portal/provisioning_profile.rb

```diff
@@ -265,7 +262,11 @@ def all(mac: false, xcode: false)
           end
 
           # filter out the profiles managed by xcode
-          profiles.delete_if(&:managed_by_xcode?) unless xcode
+          if xcode
+            warn('Apple API no longer returns XCode managed Provisioning Profiles')
+          else
+            profiles.delete_if(&:managed_by_xcode?)
```

Comment:
do we need to remove them if they are not returned anymore? Or is this because the test data is not updated?